assets/js/bluesky_comments.js

```
@@ -20,69 +37,110 @@ const createElementWithClass = (tag, className) => {
-
-        const data = await response.json();
-        if (
-            !data.thread ||
-            data.thread.$type !== "app.bsky.feed.defs#threadViewPost"
-        ) {
-            throw new ValidationError("Invalid thread data received");
-        }
-
-        return data.thread;
     }
 }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)